controllers/keda/scaledobject_controller.go

```diff
@@ -592,6 +603,7 @@ func (r *ScaledObjectReconciler) updatePromMetricsOnDelete(namespacedName string
 		for _, triggerType := range metricsData.triggerTypes {
 			metricscollector.DecrementTriggerTotal(triggerType)
 		}
+		metricscollector.RecordScalerPaused(namespacedName, scaledObject, false)
```

Comment:
this is not needed, right? Because we delete the ScaledObject, we don't have to track the Pause state. Or maybe am I missing anything?